pkg/manager/member/startscript/v2/pd_start_script.go

```diff
@@ -123,33 +123,34 @@ func renderPDMSStartScript(tc *v1alpha1.TidbCluster, name string) (string, error
 	m := &PDMSStartScriptModel{}
 	tcName := tc.Name
 	tcNS := tc.Namespace
-	peerServiceName := controller.PDMSMemberName(tcName, name)
+	peerServiceName := controller.PDMSPeerMemberName(tcName, name)
 
-	m.PDMSDomain = fmt.Sprintf("${PDMS_POD_NAME}.%s.%s.svc", peerServiceName, tcNS)
+	m.PDDomain = fmt.Sprintf("${PD_POD_NAME}.%s.%s.svc", peerServiceName, tcNS)
 	if tc.Spec.ClusterDomain != "" {
-		m.PDMSDomain = m.PDMSDomain + "." + tc.Spec.ClusterDomain
-	}
-
-	m.PDMSName = "${PDMS_POD_NAME}"
-	if tc.AcrossK8s() || tc.Spec.ClusterDomain != "" {
-		m.PDMSName = "${PDMS_DOMAIN}"
+		m.PDDomain = m.PDDomain + "." + tc.Spec.ClusterDomain
 	}
```

Comment:
Do we need to handle `AcrossK8s()` anymore?

Reply:
yes, maybe supported in future